### PR TITLE
feat([cribl-logstream]): Added [cribl-logstreak-pack]

### PR DESCRIPTION
**Pull request by nr-jbiggley**
## Summary

New pack for Cribl Logstream

### Screenshots

Image 2021-09-21 at 8 22 15 PM

### CODE REVIEW

File: packs/cribl_logstream/config.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name: cribl-logstream
+
+title: Cribl Logstream
+
+description: |+
```
Comment (polfliet) [inline packs/cribl_logstream/config.yml:5]:
```suggestion
description: |
```

**Review by polfliet — APPROVED** [review]
Looking great! Thanks @nr-jbiggley !
I just have 1 small suggestion

**Comment by nr-jbiggley** [comment]
Thanks @polfliet!

**Review by polfliet — APPROVED** [review]

**Review by polfliet — APPROVED** [review]

**Comment by polfliet** [comment]
I just fixed some remaining description fields with null values to empty strings, good to merge now

**Pull request merged**